packages/itwinui-css/src/alert/alert.scss

```diff
@@ -16,21 +16,22 @@
   background-color: var(--iui-color-background);
   border: 1px solid var(--_iui-alert-border-color);
   box-shadow: inset var(--iui-size-2xs) 0 0 var(--_iui-alert-border-color);
-  @media (forced-colors: active) {
-    background-color: Canvas;
-  }
+}
 
-  &.iui-sticky {
-    border-radius: 0;
-    position: sticky;
-    top: 0;
-    left: 0;
-  }
+@mixin iui-alert-sticky {
+  border-radius: 0;
+  position: sticky;
+  top: 0;
+  left: 0;
 }
 
 @mixin iui-alert-icon {
   @include iui-icon-style('m');
   margin-left: var(--iui-size-m);
+  fill: var(--_iui-alert-icon-color, var(--iui-color-icon-muted));
```

Comment:
What does this `var(x, y)` do?

Reply:
`y` becomes the default value for `x`.  So this example does `--_iui-alert-icon-color: var(--iui-color-icon-muted)`.

Reply:
Ah, neat